### PR TITLE
add initial files

### PR DESCRIPTION
**Pull request by lnbailey**

### CODE REVIEW

**Comment by zoeames** [comment]
Very nice quiz!  Functions perfectly!  Just one thing you missed from the instructions:
"Create a `tree` function that should build a pine tree out of a character in the Chrome dev tools console. It accepts a single object as an argument. The object should have two key/value pairs.".  Once you add in the object let me know and i will approve

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    console.log(str);
+  }
+}
+
+// User can either press the enter key or click a button that is labeled "Grow your tree".
+// Display an alert stating that both fields must have a value.
+
+function inputAlert() {
+  var rows = document.getElementById("height").value;
+  var leaves = document.getElementById("char").value;
```
Comment (zoeames) [inline quiz.js]: make this into an object:
var tree = {
rows:  document.getElementById("height").value,
leaves: document.getElementById("char").value
}

File: quiz.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  if (rows === "" || leaves === "") {
+    alert ("I can't grow your tree unless you tell me how tall AND what you want for leaves! Try again.");
+    }
+    else {
+    tree (rows, leaves)
```
Comment (zoeames) [inline quiz.js]: pass the object into this function:
tree(tree);

Comment (zoeames) [inline quiz.js]: this will only have 1 input - the object 

tree(tree)

File: quiz.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+// Create a tree function that accepts a single object as an argument.
+// The object has two key/value pairs from user input in a <input type="text"> field in the DOM.
+// Tree should be shown in the console.
+
+function tree() {
```
Comment (zoeames) [inline quiz.js]: now has one input
function tree(myTree){

File: quiz.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  var height = document.getElementById("height").value;
+  var char = document.getElementById("char").value;
+
+  for (var i = 0; i < height; i++) {
```
Comment (zoeames) [inline quiz.js:9]: you will need to change these variables to match the structure of the tree object

**Comment by copypastedeveloper** [comment]
looks good 👍

**Pull request merged**